src/Sylius/Bundle/ApiBundle/Resources/config/services/filters.xml

```diff
@@ -235,7 +250,7 @@
             <tag name="api_platform.filter" />
         </service>
 
-        <service id="sylius_api.translation_order_and_locale_filter" class="Sylius\Bundle\ApiBundle\Filter\Doctrine\TranslationOrderNameAndLocaleFilter" public="true">
+        <service id="sylius_api.filter.translation_order_and_locale" class="Sylius\Bundle\ApiBundle\Filter\Doctrine\TranslationOrderNameAndLocaleFilter" public="true">
```

Comment:
```suggestion
        <service id="sylius_api.doctrine.filter.translation_order_and_locale" class="Sylius\Bundle\ApiBundle\Filter\Doctrine\TranslationOrderNameAndLocaleFilter" public="true">
```
WDYT?

Reply:
Im not really convinced 🤔 we should also adjust it in other places with custom filters, lets leave it to other opinions also